### PR TITLE
Feature/add units test

### PR DESCRIPTION
**Pull request by dpocheronOnePoint**

### CODE REVIEW

File: test/get_tree_list_use_case_test.dart [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        .inject<RemoteTreeDataSource>(RemoteTreeDataSourceImpl());
+    DependecyInjection.instance
+        .inject<LocalTreeDataSource>(LocalTreeDataSourceImpl());
+
+    mockGetTreeLiseUseCase = MockGetTreeLiseUseCase();
+  });
+
+  group('getTrees', () {
+    final mockTreeList = [
+      Tree(
```
Comment (ettibo) [inline test/get_tree_list_use_case_test.dart]: Pourquoi tu n'utilises pas la methode.mock dans Trees ?

File: packages/api/test/remote_tree_data_source_test.dart [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    DependecyInjection.instance
+        .inject<MockRemoteTreeDataSource>(MockRemoteTreeDataSource());
+
+    remoteTreeDataSource =
+        DependecyInjection.instance.get<MockRemoteTreeDataSource>();
+  });
+
+  group('getTrees', () {
+    final mockTreeList = [
+      Tree(
```
Comment (ettibo) [inline packages/api/test/remote_tree_data_source_test.dart]: Trees.mock ?

Comment (dpocheronOnePoint) [reply packages/api/test/remote_tree_data_source_test.dart]: Ah yes !

File: test/get_tree_list_use_case_test.dart [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+class MockGetTreeLiseUseCase extends Mock implements GetTreeLiseUseCase {}
+
+class MockLocalStorage extends Mock implements LocalStorage {}
+
+void main() {
+  late MockGetTreeLiseUseCase mockGetTreeLiseUseCase;
+  List<Tree> listTree = [];
+
+  setUpAll(() async {
```
Comment (Hamitix) [inline test/get_tree_list_use_case_test.dart:20]: Any specific reason that you used 'setUpAll' in this test but 'setUp' in the other ?

Comment (dpocheronOnePoint) [reply test/get_tree_list_use_case_test.dart:20]: It's a mistake to the other file, setUpAll is better because it's called just one time. I update that

**Review by ettibo — APPROVED** [review]

**Pull request merged**